test/config-test.ts

```diff
@@ -1,6 +1,6 @@
-import defaultConfig, { debugConfig, default as config, host, port } from '../src/config.js';
+import defaultConfig, {debugConfig, default as config, host, port} from '../src/config.js';
```

Comment:
Hmm, why did spaces inside braces disappear? Does ESLint really cover this file?

BTW, ww should add spaces between braces ({ }) and the content that they surround.